GitUI/CommandsDialogs/RevisionDiffControl.Designer.cs

```diff
@@ -418,7 +418,7 @@ private void InitializeComponent()
             this.findInDiffToolStripMenuItem.Name = "findInDiffToolStripMenuItem";
             this.findInDiffToolStripMenuItem.ShortcutKeys = ((System.Windows.Forms.Keys)((System.Windows.Forms.Keys.Control | System.Windows.Forms.Keys.F)));
             this.findInDiffToolStripMenuItem.Size = new System.Drawing.Size(296, 22);
-            this.findInDiffToolStripMenuItem.Text = "Find";
+            this.findInDiffToolStripMenuItem.Text = "F&ind";
```

Comment:
Switch with `Show in Folder`
```suggestion
            this.findInDiffToolStripMenuItem.Text = "&Find";
```